### DISCUSSION TITLE
World Locking Tool with MRTK 3

### DISCUSSION THREAD

**Discussion by camroo29** [MixedRealityToolkit/MixedRealityToolkit-Unity#603] [Q&A]
Hi,

I was trying to use the WLT with MRTK3, installing it from MixedRealityFeatureTool. But there are tons of errors.  

Has the WLT been integrated with MRTK 3 or are there any other features offered by MRTK 3? If you have successfully integrated WLT with MRTK 3, please share the procedure.

Thanks

**Comment by Daniel4144** [comment]
I am using [this](https://github.com/Fuzzy-Door-Tech/WLT-MRTK3-Port) port of WLT with MRTK3.
It does work, but I think there are no samples that are compatible with MRTK3. So an update of the official WLT would be nice.
